pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -29,6 +29,7 @@ type Interface interface {
 	CreateOrUpdate(ctx context.Context, oc *api.OpenShiftCluster, obj *unstructured.Unstructured) error
 	Delete(ctx context.Context, oc *api.OpenShiftCluster, groupKind, namespace, name string) error
 	MustGather(ctx context.Context, oc *api.OpenShiftCluster, w io.Writer) error
+	SerialConsole(ctx context.Context, oc *api.OpenShiftCluster, w io.Writer, vmName string) error
```

Comment:
From what I can see, the serial console logic does not require any interaction with the cluster, it just talks to Azure. Hence, I don't think it makes sense to put this inside the kubeactions package; it seems we could probably put all the logic here directly into `pkg/frontend/admin_openshiftcluster_serialconsole.go`.

Note that e.g. the redeploy VM action does not have any deps on kubeactions: https://github.com/Azure/ARO-RP/blob/master/pkg/frontend/admin_openshiftcluster_redeployvm.go

Reply:
Good point. I've made that change.